packages/open-truss/src/configuration/engine-v1/Frame.tsx

```diff
@@ -73,14 +80,48 @@ function processProps({
     }
   }
 
+  // Add signals to props
+  eachComponentSignal(componentName, COMPONENTS, (propName, signalsType) => {
+    const signal = signals[propName]
+    if (signal) {
+      const result = signalsType.safeParse(signal)
+      if (result.success) {
+        newProps[propName] = signal
+      }
+    }
+  })
+
   return {
-    data,
-    config,
```

Comment:
Mostly because it wasn't being used to my knowledge, and I _think_ config is not particularly useful for components in general as it is just yaml. We'd likely want to create other helpers and pass those in to the components instead. What do you think?